runtime/handler.go

```diff
@@ -168,17 +168,15 @@ func handleForwardResponseStreamError(wroteHeader bool, marshaler Marshaler, w h
 
 func streamChunk(result proto.Message, err error) map[string]proto.Message {
 	if err != nil {
-		grpcCode := codes.Unknown
-		if s, ok := status.FromError(err); ok {
-			grpcCode = s.Code()
-		}
-		httpCode := HTTPStatusFromCode(grpcCode)
+		s, _ := status.FromError(err)
```

Comment:
Hmm. I know it should never happen, but seeing the second return ignored is weird, too. Any thoughts about something like
```go
s, ok := status.FromError(err)
if !ok {
  panic("err is no status")
}
```

Reply:
I wanted to make use of the new status handling in [gRPC-Go 1.10](https://github.com/grpc/grpc-go/commit/d50734d1d6ca477a72646f3022216ec39639f4cd) which is equivalent to the old handling, but I accept gRPC-Gateway can't enforce that users have the latest gRPC version. I will change it back.

Reply:
Oh that's a lovely function. Thanks for the pointer.